docs/source/tutorials/image.ipynb

```diff
@@ -521,18 +1256,28 @@
    "source": [
     "# Note: This cell is only for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.\n",
     "\n",
-    "highlighted_indices = [59915, 24798, 59701, 50340]  # verify these examples were found by cleanlab\n",
-    "if not all(x in ranked_label_issues for x in highlighted_indices):\n",
+    "issue_indices = (\n",
+    "    label_issues_df.index.values.tolist()\n",
+    "    + outlier_issues_df.index.values.tolist()\n",
+    "    + near_duplicate_issues_df.index.values.tolist()\n",
+    "    + dark_issues_df.index.values.tolist()\n",
+    "    + lowinfo_issues_df.index.values.tolist()\n",
+    ")\n",
+    "issue_indices = list(set(issue_indices))\n",
+    "\n",
+    "# low info, dark, near duplicate, outlier, label in order\n",
+    "highlighted_indices = (\n",
+    "    [53050] + [733, 3936] + [6838, 32110] + [14999, 43535] + [45561, 9065]\n",
+    ")  # verify these examples were found by cleanlab\n",
+    "\n",
+    "if not all(x in issue_indices for x in highlighted_indices):\n",
```

Comment:
why don't you break this test down per issue-type, rather than taking union over all issue types and indices.

Check the dark indices below show up in `dark_issues_df.index.values.tolist()` separately from the label issues, separately from the other issue types. That would be a stronger test and the code is super simple